src/Compilers/Shared/CompilerServerLogger.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.CodeAnalysis.CommandLine
     internal interface ICompilerServerLogger
     {
         bool IsLogging { get; }
-        void Log(string message);
+        void Log(string message, bool excludeFromBinlog = false);
```

Comment:
I feel like this breaks the abstraction a bit cause it's arg for a specific type of logger. I thought about this a bit more and given it's only in the msbuild task, not all the calls in the server, I think it's fine to just log everything.